### ISSUE TITLE
Install OpenFoam

### ISSUE THREAD

**Issue by Cadair** [rcgsheffield/iceberg_software#182]
Michael Osoba (F1788546)

http://www.openfoam.org/

**Comment by Cadair** [comment]
Currently this is waiting on SCOTCH being built.

**Comment by Cadair** [comment]
Ok, so getting closer. Seems that I have not compiled SCOTCH with shared libs enabled.

```
g++ -m64 -Dlinux64 -DWM_ARCH_OPTION=64 -DWM_DP -DWM_LABEL_SIZE=32 -Wall -Wextra -Wold-style-cast -Wnon-virtual-dtor -Wno-unused-parameter -Wno-invalid-offsetof -O3  -DNoRepository -ftemplate-depth-100 -DOMPI_SKIP_MPICXX -I/usr/local/mpi/gcc/openmpi/1.10.0/include -pthread -I/include -I/usr/local/packages6/libs/gcc/5.2/scotch/6.0.4//include -I/usr/include/scotch -I../decompositionMethods/lnInclude -IlnInclude -I. -I/usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/src/OpenFOAM/lnInclude -I/usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/src/OSspecific/POSIX/lnInclude   -fPIC -shared -Xlinker --add-needed -Xlinker --no-as-needed /usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/platforms/linux64GccDPInt32Opt/src/parallel/decompose/scotchDecomp/scotchDecomp.o -L/usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/platforms/linux64GccDPInt32Opt/lib \
            -L/lib -L/usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/ThirdParty-3.0.0/platforms/linux64GccDPInt32Opt/lib -lscotch -lscotcherrexit -lrt  -o /usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/platforms/linux64GccDPInt32Opt/lib/libscotchDecomp.so
/usr/bin/ld: /usr/local/packages6/libs/gcc/5.2/scotch/6.0.4/lib/libscotch.a(library_arch.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/local/packages6/libs/gcc/5.2/scotch/6.0.4/lib/libscotch.a: could not read symbols: Bad value
collect2: error: ld returned 1 exit status
make: *** [/usr/local/packages6/apps/gcc/5.2/openfoam/3.0.0/OpenFOAM-3.0.0/platforms/linux64GccDPInt32Opt/lib/libscotchDecomp.so] Error 1
```

**Comment by Cadair** [comment]
I think I have made this work. Documentation incoming.

**Comment by Cadair** [comment]
closed by #207
